pyEpiabm/pyEpiabm/cell.py

```diff
@@ -42,3 +45,24 @@ def enqueue_person(self, person: Person):
         :type person: Person
         """
         self.person_queue.put(person)
+
+    def _setup(self) -> None:
+        """Setup method. Should be called once Population has been setup.
+        Called by population (DOESN'T NEED TO BE CALLED MANUALLY)
```

Comment:
```suggestion
        Called by population (DOESN'T NEED TO BE CALLED MANUALLY).
```

Reply:
Might remove caps here too as it it won't look good on the online docs